core/integration/pipeline_test.go

```diff
@@ -77,6 +77,9 @@ func TestPipeline(t *testing.T) {
 		_, err = client.Sync(ctx)
 		require.NoError(t, err)
 
+		_, err = srv.Stop(ctx)
```

Comment:
The original point of the test is to show that we don't need to call this, so don't want that intention to be lost.

```suggestion
		_, err = srv.Stop(ctx) // FIXME: shouldn't need this, but test is flaking
```